doc/docs/Mode_Decomposition.md

```diff
@@ -180,61 +176,64 @@ These functions are implemented in [src/dft.cpp](https://github.com/NanoComp/mee
 
 ## How Mode Decomposition Works
 
-The theoretical basis of the mode-decomposition algorithm is the orthogonality relation satisfied by the normal modes:
+The theoretical basis of the mode-decomposition algorithm is an orthogonality relation satisfied by the eigenmodes:
 
-$$ \left\langle \mathbf{E}_m^{\sigma} \right|
-   \left.       \mathbf{H}^\tau_n     \right\rangle
-   =C_{m}\delta_{mn}\delta_{\sigma\tau} 
+$$
+\left\langle \psi_m^\sigma, \psi_m^\tau_n \right\rangle
```

Comment:
This equation is not being rendered correctly due to some missing braces for the superscript. It should be changed to:
```
\left\langle \psi_m^{\sigma}, \psi_m^{\tau_n} \right\rangle
```

Reply:
Also, there are a couple of typos in this equation, it should be changed to: `\left\langle \psi_m^{\sigma}, \psi_n^{\tau} \right\rangle`